edu.cuny.hunter.log.tests/.classpath

```diff
@@ -6,10 +6,6 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="test cases/">
-		<attributes>
-			<attribute name="test" value="true"/>
-		</attributes>
-	</classpathentry>
+	<classpathentry kind="src" path="test cases/"/>
```

Comment:
That is the purpose of this change?

Reply:
Eclipse updated the class path after I bumped up the core package version

Reply:
I don't see a reason for this change. Can you investigate why Eclipse is doing this? Is it because this attribute is deprecated? But also, and perhaps more importantly, the test plugin is not included in the distribution and thus should not be part of the release.

Reply:
The test plugin is not included in the release. I only updated it's dependency version

Reply:
Were the test attribute originally added for used with JUnit?

Reply:
Unit does not need an explicit test attribute as the Eclipse JUnit integration has no special classpath configuration. It will work off the output folders defined in your classpath and should find all classes compiled to those folders